src/title-nav/block.spec.js

```diff
@@ -0,0 +1,144 @@
+// title-nav test
+jest.setTimeout(30000)
+import {
+    createNewPost,
+    enablePageDialogAccept,
+    getEditedPostContent,
+    insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+import {
+    clickElementByText,
+    blockStartup,
+    updateRangeInput,
+    clickCheckbox,
+    openSidebarPanelWithTitle
+} from '../test-helpers'
+
+const block = {blockTitle: 'Titled Navigation', blockName: 'purdue-blocks/title-nav'}
+
+describe( 'Titled Navigation Block', () => {
+    beforeAll( async () => {
+        await enablePageDialogAccept();
+    } );
+    beforeEach( async () => {
+        await createNewPost();
+    } );
+
+    it( 'Block should be available.', async () => {
+        await insertBlock( 'Titled Navigation' )
+
+
+        // tests that the block is properly inserted and matches the existing snapshot
+        expect(await page.$('[data-type="purdue-blocks/title-nav"]')).not.toBeNull()
+        expect( await getEditedPostContent() ).toMatchSnapshot()
+    } )
+
+    it( 'Navigation links added correctly.', async () => {
```

Comment:
maybe nest another describe block here for the navigation to again make the it blocks more friendly and a little more organized. 

perhaps something like this
```
describe ('Navigation', ()=> {
  it('should add links correctly.' , async () => { 
   ... 
});
})